src/Symfony/Component/Mime/NamedAddress.php

```diff
@@ -16,6 +16,8 @@
  */
 final class NamedAddress extends Address
 {
+    const FROM_STRING_PATTERN = '~(?<name>.*)<(?<address>[^>]+)>~';
```

Comment:
I think the RFC for an address is more complex than this one. We should at least support the most common forms.

Reply:
You mean the inverse of `Rfc2231Encoder`?